core/src/main/java/uk/gov/justice/services/core/envelope/LoggingValidationExceptionHandler.java

```diff
@@ -1,18 +1,32 @@
 package uk.gov.justice.services.core.envelope;
 
+import static java.lang.String.format;
+
+import uk.gov.justice.services.core.json.DefaultJsonValidationLoggerHelper;
+
 import javax.enterprise.inject.Alternative;
-import javax.inject.Inject;
 
+import org.everit.json.schema.ValidationException;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Alternative
 public class LoggingValidationExceptionHandler implements EnvelopeValidationExceptionHandler {
 
-    @Inject
-    private Logger logger;
+    private DefaultJsonValidationLoggerHelper defaultJsonValidationLoggerHelper = new DefaultJsonValidationLoggerHelper();
```

Comment:
This can be injected as a JsonValidationLoggerHelper

Reply:
As discussed we cannot inject here as producer uses reflection. Have tried alternative methods with the BeanManager, '@Vetoed' etc but CDI expects us to use beans.xml and we don't currently do that on project.